### PR TITLE
DOCKER-73 allow test image to be configurable in helm chart

### PR DESCRIPTION
**Pull request by jCOTINEAU**
- Add Values parameter tests.image
- Update Outdated README

Please ensure your pull request adheres to the following guidelines:
- [x] explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [x] Document your Changes in the `CHANGELOG.md` file of the respected chart as well as the `Chart.yaml`
- [x] Bump the Version number of the respected chart

### CODE REVIEW

**Review by carminevassallo — CHANGES_REQUESTED** [review]
Thanks for spotting the wrong usage  of `tests.image`! This PR can be both viewed as feature enhancement and bug fix but at least originally this was meant to be the former, so I suggest that we increment the minor instead of patch version. I've tested locally, the generated k8s are the same as before, I added a few small comments and probably we should replace the deprecated image. @leo-geoffroy-sonarsource, could you please give a second review?

**Comment by leo-geoffroy-sonarsource** [comment]
Nitpick - You can remove the colon ":" in your commit message, as a general rule we just keep space between ticket number and comment

**Comment by leo-geoffroy-sonarsource** [comment]
> Thanks for spotting the wrong usage of `tests.image`! This PR can be both viewed as feature enhancement and bug fix but at least originally this was meant to be the former, so I suggest that we increment the minor instead of patch version. I've tested locally, the generated k8s are the same as before, I added a few small comments and probably we should replace the deprecated image. @leo-geoffroy-sonarsource, could you please give a second review?

Agreed with the minor version increase, it is indeed an improvement
Good spot on the deprecated image minideb-extras, and I suggest opening a different task for that as this will requires a bit of investigation/testing (I'm curious about the functional differences between minideb-extras and minideb)

File: charts/sonarqube-dce/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sonarqube-dce
 description: SonarQube offers Code Quality and Code Security analysis for up to 27 languages. Find Bugs, Vulnerabilities, Security Hotspots and Code Smells throughout your workflow.
 type: application
-version: 4.2.1
+version: 4.2.2
```
[the remark this answers is not in the document]
Comment (leo-geoffroy-sonarsource) [reply charts/sonarqube-dce/Chart.yaml:5]: See [my comment](https://github.com/SonarSource/helm-chart-sonarqube/pull/259#issuecomment-1260523773)

**Comment by jCOTINEAU** [comment]
> > Thanks for spotting the wrong usage of `tests.image`! This PR can be both viewed as feature enhancement and bug fix but at least originally this was meant to be the former, so I suggest that we increment the minor instead of patch version. I've tested locally, the generated k8s are the same as before, I added a few small comments and probably we should replace the deprecated image. @leo-geoffroy-sonarsource, could you please give a second review?
> 
> Agreed with the minor version increase, it is indeed an improvement Good spot on the deprecated image minideb-extras, and I suggest opening a different task for that as this will requires a bit of investigation/testing (I'm curious about the functional differences between minideb-extras and minideb)

I think we could ship the minideb-extras into minideb change, as the only purpose is running a curl command, and it is directly tested by the CI process.

What do you think ?

File: charts/sonarqube-dce/values.yaml [hunk 2]
```diff
@@ -518,8 +518,10 @@ logging:
   jsonOutput: false
 
 tests:
+  image: bitnami/minideb-extras
```
[the remark this answers is not in the document]
Comment (jCOTINEAU) [reply charts/sonarqube-dce/values.yaml:521]: resolving with [this](https://github.com/SonarSource/helm-chart-sonarqube/pull/259#issuecomment-1260641671), adding a ticket

**Review by carminevassallo — APPROVED** [review]

**Comment by carminevassallo** [comment]
> Well the minideb does not contain curl so definitely a scope for another tickets

oh ok! can you please create one (if you haven't done it already)? Apart from that, I'm fine with the changes, what about you @leo-geoffroy-sonarsource ?

**Review by leo-geoffroy-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**